ksql-common/src/test/java/io/confluent/ksql/GenericRowTest.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2017 Confluent Inc.
```

Comment:
nit: 2018

Reply:
Should we change this for all the code? We had a discussion on this before.

Reply:
Not according to the lawyers. The copyright should be the year it was written. So new files should go in with 2018, but old ones should keep the year they have.